README.md

```diff
@@ -35,23 +35,27 @@ You can also chat with us in [Slack](https://homebridge-slackin.glitch.me).
 
 ## Installation
 
+The [Homebridge Wiki](https://github.com/nfarina/homebridge/wiki) contains step-by-step instruction on how to install Node.js and setup Homebridge as a service so it automatically start on boot:
+
+* [Setup Homebridge on a Raspberry Pi (Raspbian)](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Raspbian)
+* [Setup Homebridge on Debian/Ubuntu Linux](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Debian-or-Ubuntu-Linux)
+* [Setup Homebridge on macOS](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-macOS)
+* [Setup Homebridge on Windows 10](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Windows-10)
+* [Setup Homebridge on Docker (Linux)](https://github.com/nfarina/homebridge/wiki/Install-Homebridge-on-Docker)
+* [Other Platforms](https://github.com/nfarina/homebridge/wiki/Other-Platforms)
+
 #### Quick Overview
-1. **Node v4.3.2 or greater is required.** Check by running: `node --version`. The plugins you use may require newer versions.
-2. **On Linux only:** Install the libavahi-compat-libdnssd-dev package: `sudo apt-get install libavahi-compat-libdnssd-dev`
-3. Install Homebridge using: `npm install -g homebridge` _or_ `sudo npm install -g --unsafe-perm homebridge` (see below)
-4. Install the plugins using: `npm install -g <plugin-name>`
-5. Create the `config.json` file.
 
-**Note:** If you're running on a Raspberry Pi, you should have a look at the [Wiki](https://github.com/nfarina/homebridge/wiki/Running-HomeBridge-on-a-Raspberry-Pi).
+1. **Node v4.3.2 or greater is required.** Check by running: `node -v`. The plugins you use may require newer versions.
+2. Install Homebridge using: `npm install -g homebridge`
+3. Install the plugins using: `npm install -g <plugin-name>`
+4. Create the `config.json` file.
 
 #### Installation Details
-Homebridge is published through [NPM](https://www.npmjs.com/package/homebridge) and should be installed "globally" by typing:
 
-    sudo npm install -g --unsafe-perm homebridge
```

Comment:
Installing with `--unsafe-perm` is no longer required since Homebridge no longer includes any modules that have c++ dependencies / on install scripts.